src/sync/order_rewards.py

```diff
@@ -1,19 +1,17 @@
 """Main Entry point for app_hash sync"""
 import csv
-import logging.config
 import os.path
 
 from dune_client.file.interface import FileIO
 from pandas import DataFrame
 
 from src.fetch.orderbook import OrderbookFetcher
+from src.logger import set_log
 from src.models.block_range import BlockRange
 from src.sync.common import last_sync_block, aws_login_and_upload
 from src.sync.config import SyncConfig
 
-log = logging.getLogger(__name__)
-logging.basicConfig(format="%(asctime)s %(levelname)s %(name)s %(message)s")
```

Comment:
Looks like this formatting was used a lot and I don't see any place where `logging` initializes this format. Is that intended?

Reply:
Turns out this formatting is the default and so I don't see any need to set it (sorry I didn't mention it).